applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py

```diff
@@ -387,6 +388,7 @@ def AssembleTestSuites():
     validationSuite.addTest(TShellQ4ThinLinearDynamicTests('test_execution'))
     validationSuite.addTest(TShellQ4ThinNonLinearDynamicTests('test_execution'))
     validationSuite.addTest(TShellQ4ThinOrthotropicLaminateLinearStaticTests('test_execution'))
+    validationSuite.addTest(TPeriodicLinearTest('test_execution'))
```

Comment:
how long does this test run on our machine?

Reply:
2-3 seconds in release mode.